doc/realtime_servo/realtime_servo_tutorial.rst

```diff
@@ -1,134 +1,123 @@
+    ros2 launch moveit2_tutorials servo_cpp_interface_demo.launch.py
 
-Robot Requirements
-------------------
-The servo node streams an array of position or velocity commands to the robot controller. This is compatible with ros\_control ``position_controllers/JointGroupPositionControllers`` or ``velocity_controllers/JointGroupVelocityControllers``. You can check if these controllers are available for your robot by searching for the controller config file (typically named ``controllers.yaml``). After launching the robot, you can check if any ros_control controllers are available with: ::
+An Rviz window should appear with a Panda arm and collision object. The arm will joint-jog for a few seconds before switching to a Cartesian movement. As the arm approaches the collision object, it slows and stops.
 
-    rosservice call /controller_manager/list_controllers
+Expected Output
+---------------
+.. raw:: html
 
-And switch to the desired controller with: ::
+    <video width="700px" nocontrols="true" autoplay="true" loop="true">
+        <source src="../../_static/C++_Interface_Demo.webm" type="video/webm">
+        Joint and Cartesian jogging with collision prevention
+    </video>
 
-    rosservice call /controller_manager/switch_controllers controller_to_start controller_to_stop
+Entire Code
+-----------
+The entire code is available :codedir:`here<realtime_servo/src/servo_cpp_interface_demo.cpp>`
 
-**NOTE:** You can tab-complete to help fill these commands.
+.. tutorial-formatter:: ./src/servo_cpp_interface_demo.cpp
 
-Servoing may work on other robots that have a different control scheme but there is no guarantee. It has been tested heavily on UR robots using the `ur_modern_driver <https://github.com/ros-industrial/ur_modern_driver>`_. The servo node currently does not limit joint jerk so may not be compatible with most heavy industrial robots.
 
-The servo node can publish ``trajectory_msgs/JointTrajectory`` or ``std_msgs/Float64MultiArray`` message types. This is configured in a yaml file (see ``config/ur_simulated_config.yaml`` for an example). Most robots that use ros_control will use the Float64MultiArray type. Some UR robots using older driver versions can require the JointTrajectory message type.
+Servo Overview
+--------------
 
-ROS Signals
------------
-An `rqt_graph` of the servo node is shown below (Enlarge by clicking it). Most of these connections can be ignored. The important ones are:
+MoveIt Servo allows you to stream End Effector (EEF) velocity commands to your manipulator and have it execute them concurrently. This enables teleoperation via a wide range of input schemes, or for other autonomous software to control the robot - in visual servoing or closed loop position control for instance.
```

Comment:
It's weird that this overview is halfway through the document. Recommend merging it with the first section.